docs/Documentation-CHN/UserGuide/5-Operation Manual/4-SQL Reference.md

```diff
@@ -240,17 +240,17 @@ Condition  : <Expression> [(AND | OR) <Expression>]*
 Expression : [NOT | !]? <TimeExpr> | [NOT | !]? <SensorExpr>
 TimeExpr : TIME PrecedenceEqualOperator <TimeValue>
 SensorExpr : (<Timeseries> | <Path>) PrecedenceEqualOperator <PointValue>
-GroupByClause : LPAREN <TimeUnit> (COMMA TimeValue)? COMMA <TimeInterval> (COMMA <TimeInterval>)* RPAREN
+GroupByClause : LPAREN <TimeInterval> COMMA <TimeUnit> (COMMA <TimeUnit>)? RPAREN
+TimeInterval: '[' TimeUnit ',' TimeUnit ']'
```

Comment:
`TimeValue`, not `TimeUnit`.

Reply:
I think it's TimeUnit, not TimeValue.
TimeUnit is like '3h', '1d' and so on. However, TimeValue is just a timestamp.
Here, obviously, we need the time unit instead of timeValue.

Reply:
But I thought `TimeInterval` is [TimeValue, TimeValue].